.github/workflows/README.md

```diff
@@ -5,6 +5,152 @@ CI system guide
 <tr><th></th><td></td>
 -->
 
+# Introduction
+
+This is the documentation of CI system that is used to build and publish ArcticDB. \
```

Comment:
I don't think `\` are needed. AFAIK leaving a new line will not do a line break. Leaving two new lines should do a line break.

Reply:
Yes, I want a new line here.
I prefer new sentences to be on new lines as it makes it easier to read.